Ladybird/Android/src/main/res/layout/activity_main.xml

```diff
@@ -4,16 +4,14 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    tools:context=".LadybirdActivity">
 
-    <TextView
-        android:id="@+id/sample_text"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+    <org.serenityos.ladybird.WebView xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:id="@+id/web_view"
```

Comment:
You don't have to repeat `xmlns` on every element; it's enough to do it on the root one.

I don't think you want the `ConstraintLayout` here; something like `CoordinatorLayout` + `AppBarLayout` for a header bar + `SwipeRefreshLayout` wrapping the `WebView` would be more suitable

Reply:
Sure it probably isn't right. I put zero effort into the views aside from replacing the template TextView with my view.

Reply:
(In case it's not clear, I'm trying to focus on getting *something* on screen first before making the app look good)

Reply:
(Sure, that's reasonable.)